easybuild/easyconfigs/p/Pango/Pango-1.42.4-fosscuda-2018b.eb

```diff
@@ -0,0 +1,36 @@
+easyblock = 'ConfigureMake'
+
+name = 'Pango'
+version = '1.42.4'
+
+homepage = 'http://www.pango.org/'
+description = """Pango is a library for laying out and rendering of text, with an emphasis on internationalization.
+Pango can be used anywhere that text layout is needed, though most of the work on Pango so far has been done in the
+context of the GTK+ widget toolkit. Pango forms the core of text and font handling for GTK+-2.x."""
+
+toolchain = {'name': 'fosscuda', 'version': '2018b'}
+
+source_urls = [FTPGNOME_SOURCE]
+sources = [SOURCELOWER_TAR_XZ]
+checksums = ['1d2b74cd63e8bd41961f2f8d952355aa0f9be6002b52c8aa7699d9f5da597c9d']
+
+builddependencies = [
+    ('GObject-Introspection', '1.54.1', '-Python-2.7.15'),
+    ('pkg-config', '0.29.2'),
+]
+
+dependencies = [
+    ('X11', '20180604'),
+    ('GLib', '2.54.3'),
+    ('cairo', '1.14.12'),
+    ('HarfBuzz', '1.9.0'),
```

Comment:
@akesandgren Any specific reason you used this old `HarfBuzz` version (see also failing test)?

Reply:
I don't remember. I think there was something I was building that didn't work with HarfBuzz 2 but right now I can't find it.